relayer/src/light_client/tendermint.rs

```diff
@@ -1,149 +1,128 @@
 use std::convert::TryFrom;
 
+use tendermint_rpc as rpc;
+
 use tendermint_light_client::{
-    builder::LightClientBuilder, builder::SupervisorBuilder, light_client, store, supervisor,
-    supervisor::Handle, supervisor::Supervisor, types::Height as TMHeight, types::LightBlock,
+    components::{self, io::AtHeight},
+    light_client::{LightClient as TmLightClient, Options as TmOptions},
+    operations,
+    state::State as LightClientState,
+    store::{memory::MemoryStore, LightStore},
+    types::Height as TMHeight,
```

Comment:
nit: why don't we get a clippy on `TMHeight`?

Reply:
Because it's a type alias (I think), will fix it in #793.